src/Cmfd.cpp

```diff
@@ -1872,71 +1476,71 @@ void Cmfd::generateKNearestStencils(){
  * @param fsr The fsr being updated.
  * @return the ratio used to update the FSR flux.
  */
-FP_PRECISION Cmfd::getUpdateRatio(int cell, int group, int fsr){
+FP_PRECISION Cmfd::getUpdateRatio(int cell, int group, int fsr) {
 
   FP_PRECISION ratio = 0.0;
   FP_PRECISION total_distance;
   std::vector< std::pair<int, FP_PRECISION> >::iterator iter;
 
-  if (_centroid_update_on){
+  if (_centroid_update_on) {
 
     total_distance = _centroid_total_distances[fsr];
```

Comment:
I'll change the second value in the pair to `(1.0 - cell_distance / total_distance)`.